configure.ac

```diff
@@ -746,6 +758,7 @@ echo "	yescrypt passwords encryption:	$with_yescrypt"
 echo "	nscd support:			$with_nscd"
 echo "	sssd support:			$with_sssd"
 echo "	subordinate IDs support:	$enable_subids"
+echo "	enable lastlog:		$enable_lastlog"
```

Comment:
The spacing here doesn't look right (even when I wget the patch and remove the first lines).  I think you need to remove the space after the :, and add two tabs.

Reply:
There were 2 tabs, and I think another one was missing. Please check it on your setup and if it works close this conversation.

Reply:
I checked out now and spacing is correct now.